.github/workflows/integration-test.yml

```diff
@@ -54,22 +54,18 @@ on:
           # Default ZenML deployments
           - default
           - default-docker-orchestrator
-          - default-kubeflow-orchestrator
```

Comment:
You also need to remove this from the integration test matrix in the CI github workflow to get the CI to pass.